File: exercicios/ordenacao3.c
#include <stdio.h>

int main(){
    float n1, n2, n3;

    printf("Digite um número: ");
    scanf("%f", &n1);
    printf("Digite outro número: ");
    scanf("%f", &n2);
    printf("Digite o ultimo número: ");
    scanf("%f", &n3);

    if (n1 < n2 && n1 < n3){
        if (n2 < n3){
            printf("%f, %f, %f \n", n1, n2, n3);
        } else{
            printf("%f, %f, %f \n", n1, n3, n2);
        }  
    }
    if (n2 < n1 && n2 < n3){
        if (n1 < n3){
            printf("%f, %f, %f \n", n2, n1, n3);
        } else{
            printf("%f, %f, %f \n", n2, n3, n1);
        }
    }
    if (n3 < n1 && n3 < n2){
        if (n1 < n2){
            printf("%f, %f, %f \n", n3, n1, n2);
        } else{
            printf("%f, %f, %f \n", n3, n2, n1);
        }
    }
    if (n1 == n2 && n2 == n3){
        printf("Todos são iguais");
    }
    
    return 0;
}

File: exercicios/python/ordenacao3.py
n1 = float(input("Digite um número: "))
n2 = float(input("Digite outro número: "))
n3 = float(input("Digite o ultimo número: "))

if n1 < n2 and n1 < n3:
    if n2 < n3:
        print(n1, n2, n3)
    elif n3 < n2:
        print(n1, n3, n2)
if n2 < n1 and n2 < n3:
    if n1 < n3:
        print(n2, n1, n3)
    elif n3 < n2:
        print(n2, n3, n1)
if n3 < n2 and n3 < n1:
    if n1 < n2:
        print(n3, n1, n2)
    elif n2 < n1:
        print(n3, n2, n1)
if n1 == n2 and n2 == n3:
    print("Todos são iguais")